packages/dataclass-editor/src/components/control/Control.tsx

```diff
@@ -0,0 +1,22 @@
+import { Flex, Separator } from '@axonivy/ui-components';
+import type { ReactElement } from 'react';
+import { Fragment } from 'react';
+
+type ControlProps = {
+  buttons: Array<ReactElement>;
+};
+
+export const Control = ({ buttons }: ControlProps) => {
+  const controlButtons = buttons.map((button, index) => {
+    if (index === 0) {
+      return button;
+    }
+    return (
+      <Fragment key={index}>
+        <Separator decorative orientation='vertical' style={{ height: '20px', margin: '0 10px' }} />
+        {button}
+      </Fragment>
+    );
+  });
+  return <Flex>{controlButtons}</Flex>;
```

Comment:
why not simply use gap instead of this complicated margin stuff?

Reply:
The smallest gap or even no gap is already too big.

Reply:
as metioned inanother comment (https://github.com/axonivy/dataclass-editor-client/pull/4#discussion_r1734074894) just remove the margin from the sepeator does the trick ;)